x/gamm/twap/store.go

```diff
@@ -95,7 +95,7 @@ func (k Keeper) storeNewRecord(ctx sdk.Context, twap types.TwapRecord) {
 // returns the TWAP record from state for (id, t', asset0, asset1),
 // where t' is such that:
 // * t' <= t
-// * there exists no t'' in state, where t'' < t'
+// * there exists no t'' in state, where t' < t''
```

Comment:
Wasn't 100% sure on this but it seems to be the case

Reply:
The old comment is more directionally correct, but still has a bug. It should read:
```suggestion
// * there exists no `t'' <= t` in state, where `t' < t''`
```

We return `t' < t` and the guarantee is there is no other state record with a time greater than t' but less than t (namely that t' is the closest to t, disallowing entries > t)